book/website/community-handbook/communication-channels.md

```diff
@@ -1,9 +1,88 @@
+
+To streamline communication, issues that seek input or require voting are labelled appropriately, such as [`community`](https://github.com/the-turing-way/the-turing-way/labels/community), when a certain discussion requires community input.
+When soliciting feedback, adding clear titles, mentioning the right people and indicating timelines help improve the engagement.
+These issues can be crossposted on other communication channels to boost their visibility.
+
+Some examples of when issues and discussions can be used for communication purposes are the following:
+- Inviting community inputs, ideas and feedback: Issue #2811 about [Using/supporting open infrastructure - suggested alternatives to currently used tools](https://github.com/the-turing-way/the-turing-way/issues/2811)
+- Keeping a record of an ongoing project: Discussion #3081 about [Technical Improvements to enhance the usability of _The Turing Way_ Book (GSoC 2023)](https://github.com/the-turing-way/the-turing-way/discussions/3081)
+
+To indicate the specific nature of the issue/discussion, we also use words such as `[TALK]`, `[INVITATION]` and `[FEEDBACK NEEDED]` (in square brackets) before writing the full title.
+
+## Synchronous Communications
+
+We use the Slack and social media platforms to engage with our collaborators and community members synchrnously.
+
+### Slack Workspace
+
+_The Turing Way_ Slack Workspace (theturingway.slack.com) is one of our core communication channels. 
+This platform enables instant messaging, fostering quick exchanges and real-time collaborations among team members.
+It is a space where folks ask questions, learn about each other’s work, and get involved with _The Turing Way_ and other projects. 
+While it is not the only communication channel used by the community, it is a popular platform, increasingly used by remote teams around the world.
+
+You can join the workspace using the [joining invitation](https://theturingway.slack.com/join/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw#/shared-invite/email).
+
+### Social media channels
+
+In expanding our reach beyond project-specific communication channels, _The Turing Way_ project delivery team uses social media platforms to connect with a wider audience and share project updates broadly.
+
+- **Twitter (@turingway):** Used since 2020, our Twitter presence has played an important role in getting the community engaged in real-time with updates and announcements from the project. Follow [@turingway](https://twitter.com/turingway).
+- **Fosstodon (@turingway):** Since 2023, we have been using [Fosstodon](https://fosstodon.org/@turingway), a platform for open science practitioners in our community.
+- **LinkedIn Page:** Since late 2023, we have been testing and expanding our presence on [LinkedIn](https://www.linkedin.com/company/the-turing-way) among professional networks.
+
```

Comment:
Maybe we could add a note here about how a Gitter channel was established, but has since been retired/not been maintained?

Reply:
Please add that.

Reply:
Added!